modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {applicationsMenuPageTest} from '../../fixtures/applicationsMenuPageTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {getRandomString} from '../../utils/util';
+import {journalPagesTest} from './fixtures/journalPagesTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	applicationsMenuPageTest,
+	loginTest,
+	journalPagesTest
+);
+
+const PERMISSIONS_LOCATORS = [
+	'#guest_ACTION_DELETE',
+	'#guest_ACTION_PERMISSIONS',
+];
+
+test('LPD-17782: This is a test for bulk permissions of web content', async ({
+	apiHelpers,
+	journalEditArticlePage,
+	journalPage,
+	page,
+}) => {
+	await apiHelpers.featureFlag.updateFeatureFlag('LPD-16469', true);
```

Comment:
We can use [`featureFlagsTest`](https://github.com/liferay-content-management/liferay-portal/blob/8eb43143c0430e665190610ed7d120e3dd05a52a/modules/test/playwright/tests/frontend-data-set-views-web/itemActions.spec.ts#L17-L20), instead of use directly (this will be reset the FF to the original state after run the test).

Reply:
Didn't know about it, thanks!